.circleci/config.yml

```diff
@@ -18,40 +18,51 @@ jobs:
           name: run pre-commit
           command: ~/.local/bin/pre-commit run --all-files
 
-  deploy:
-    docker:
-      - image: nikolaik/python-nodejs:python3.7-nodejs12
+  deploy-staging:
+    machine: 
+      image: ubuntu-2004:202010-01
     working_directory: ~/covid-api
     steps:
       - checkout
-      - setup_remote_docker
+      - run: 
+          name: use python 3
+          command: |
+            pyenv versions
```

Comment:
Guessing this was in here for debugging?

Reply:
The `pyenv versions` command was for debugging, the next one `pyenv global 3.8.5` is necessary because the linux VM that the pipeline runs in uses python2 by default

Reply:
Yeah that's cool, was only referring to the `versions` one 💯